### DISCUSSION TITLE
Unrealistic snow depth in non-glaciated Arctic regions

### DISCUSSION THREAD

**Discussion by AdrienDams** [ESCOMP/CTSM#1893] [Q&A] [answered]
Dear CTSM community,

Heidrun Matthes and I have been working on a CTSM evaluation study over the Arctic for the past year. We are using a "home-made" irregular domain and ERA5 data as forcing

For this first run, we have made a "regular" spin-up using ERA5 1980 year 15 times. Looking to the logs, the file was used to set the surface datasets is `/lnd/clm2/initdata_map/clmi.I2000Clm50BgcCrop.2011-01-01.1.9x2.5_gx1v7_gl4_simyr2000_c190312.nc` (why is it a BGC run for a SP compset?).

We may have found a problem with the snow depth represented by the model over some Arctic regions. In the first figure below, you can see snow depth values (daily average of 22-02-1980) greater than 8m in the northern part of the Taymir Peninsula, and values greater than 12, 16m or more in the Sevemaya, Svalbard and Ellesmere islands, respectively.

![042_snow_depth_1980-02-22](https://user-images.githubusercontent.com/24732655/200880820-73b7aced-38ac-4852-8311-e8a6c8d22e6d.png)

We know that some of these areas are considered as glaciers by the model. However, if you look at the second figure bellow (edit: PCT_GALCIER variable over the domain), you can see that even in areas that are not considered glaciers, there are very large snow depth values in Figure 1.

![glacier_PCT](https://user-images.githubusercontent.com/24732655/200880866-b6bf5316-7f61-41ac-8b33-86bd4e0e1869.png)

When we compare the results of the ground temperature model to stations in these areas, we find a very large positive anomaly (model too warm) which could be explained by this unrealistically deep snow cover.

It is important to note that we did not find this contrast with a similar evaluation study using CLM4.5 (in press).

Have any of you encountered this same observation? Do you have any recommendations for us to resolve this issue?

I'm sorry about the on-the-fly mapping. I have attached the env_run.xml and user_nl_clm files from this run.

[env_run.xml.txt](https://github.com/ESCOMP/CTSM/files/9972882/env_run.xml.txt)
[user_nl_clm.txt](https://github.com/ESCOMP/CTSM/files/9972883/user_nl_clm.txt)

**Comment by AdrienDams** [comment]
I made a plot of the snow depth from `/lnd/clm2/initdata_map/clmi.I2000Clm50BgcCrop.2011-01-01.1.9x2.5_gx1v7_gl4_simyr2000_c190312.nc`. As suggested by Keith, it looks like there is deep snow cover in most of the areas (sorry for the approximate mapping, these files are not very easy to use for me).

![initfile_snow_cover](https://user-images.githubusercontent.com/24732655/200884611-3c554794-1cd1-4808-92dd-82ecbfaed3da.png)

**Comment by olyson** [comment]
One problem we see with some of our initial files is very deep snow in the Canadian Arctic, for example.  I don't see that here however.  You could try spinning up from a cold start where there is no snow cover over vegetated areas.  Or you could zero out the snow cover for vegetated landunits in the initial file that is being used for the spinup.
I don't think we have much experience with CLM forced by ERA5.  We usually use GSWP3V1.  Shown below are annual mean results for snow depth from a 5 year 1deg run with CTSM.  The initial file used is the same as the one you used for your spinup.  There are some similarities (e.g., Ellesmere Island) but also some differences (southern Alaska, Novaya Zemlya), so forcing is playing a role here.
I don't understand the difference between what is plotted in the first two figures, other than the units.

![SNOWDP_CTSM51D100](https://user-images.githubusercontent.com/18671367/200929511-c83eaec9-48f9-43ef-a225-5b590e0bf627.png)
![SNOWDP_CTSM51D100_colorbar](https://user-images.githubusercontent.com/18671367/200929537-36d21236-659f-4940-b004-4e92623e9cc6.png)

**Reply by AdrienDams** [reply]
I am sorry I didn't explain the second figure I shared. It's the variable PCT_GLACIER over my domain.

**Reply by AdrienDams** [reply]
Can I ask you @olyson which forcing year do you use for this simulation? Is it starting from year 2000?

**Reply by olyson** [reply]
That plot is an average of years 1996-2000 (GSWP3V1 atm forcing) from a simulation initialized in 1991 using ctsm5.1.dev100 (--compset 2000_DATM%GSWP3v1_CLM51%BGC-CROP_SICE_SOCN_MOSART_SGLC_SWAV --res f09_g17).

**Comment by billsacks** [comment]
Regarding the difference between CLM4.5 and CLM5.0 or later: In CLM4.5 and earlier, we imposed a maximum snow water equivalent of 1 m. Starting with CLM5.0, we increased this to 10 m to allow for more realistic snow behavior over glacier regions. To avoid an artificial difference between glacier and non-glacier columns, we consistently use the 10 m max everywhere. As you have found, though, this can lead to very large snow packs if there are small biases that accumulate over time. In CLM4.5, once the snow pack reached 1 m water equivalent, any additional snow would be sent directly to the river as ice runoff; now that "snow capping" is only applied after the snow pack reaches 10 m water equivalent.

To achieve what @olyson suggests for resetting the snow pack: you can use the namelist variable "reset_snow = .true." for this purpose. The documentation of this variable states:

> If set to .true., then reset the snow pack over non-glacier columns to a small value.
This is useful when transitioning from a spinup under one set of atmospheric forcings
to a run under a different set of atmospheric forcings: By resetting too-large snow packs,
we make it more likely that points will remain only seasonally snow-covered under the new
atmospheric forcings. (This is particularly true in a coupled run, where starting with a
too-large snow pack can cool the atmosphere, thus maintaining the too-large snow pack.)
>
> WARNING: Setting this to .true. will break water conservation for approximately the first
day of the new run. This is by design: The excess snow is completely removed from the system.

However, note that the snow pack can require a few centuries to spin up in some regions.

**Reply by AdrienDams** [reply]
First thank you very much @olyson and @billsacks for your help and this precious information. I have more questions related to that.

- I will try to use this namelist variable and see the results. Yet, we are not 100% sure that this will solve our problem. Because even in the grid points where the initial snow depth is less than 0.4 m, we see that the snow accumulates over the years and never melts. 

- Do you have an initial file like `clmi.I2000Clm50BgcCrop.2011-01-01.1.9x2.5_gx1v7_gl4_simyr2000_c190312.nc` but with the namelist variable "reset_snow = .true." turned on by any chance? This will save us a huge computation time because we won't have enough resources to make a centuries spin-up in my project.

- Would it be possible to impose a SWE of 1m in non-glacier points like in CLM4.5? If not, can we cap SWE at 1m everywhere in CLM5?

**Reply by billsacks** [reply]
> I will try to use this namelist variable and see the results. Yet, we are not 100% sure that this will solve our problem. Because even in the grid points where the initial snow depth is less than 0.4 m, we see that the snow accumulates over the years and never melts.

I agree: it's quite possible this won't solve your problem. It is really just designed to solve the problem of multiple stable states in the snow pack - that once you start with a large snow pack, it can be self-sustaining even if the forcings wouldn't have created that snow pack in the first place.

> Do you have an initial file like clmi.I2000Clm50BgcCrop.2011-01-01.1.9x2.5_gx1v7_gl4_simyr2000_c190312.nc but with the namelist variable "reset_snow = .true." turned on by any chance?

Having "reset_snow = .true." in the original creation of an initial conditions file is not very relevant: this variable is more for switching from one set of forcings to another. Our spin-ups generally originate from cold start conditions, which already involve starting with little or no snow pack.

> This will save us a huge computation time because we won't have enough resources to make a centuries spin-up in my project.

I should have clarified that the centuries-long spinup is only important for the snow pack if you care about the snow pack itself or if you are coupled to an ocean model where you want to avoid transient behavior. For most purposes, a short (e.g., your 15-year) spinup is enough to get a decent representation of the snow pack.

> Would it be possible to impose a SWE of 1m in non-glacier points like in CLM4.5? If not, can we cap SWE at 1m everywhere in CLM5?

In user_nl_clm, set `h2osno_max = 1000`. This will use a SWE max of 1m for all points – glacier and non-glacier – but as long as your purpose is not to understand the snow pack / glacier dynamics, this should be fine.

**Reply by AdrienDams** [reply]
Thank you again for you quick and very helpful answer Bill!

We are not interested in glacier dynamics. However, in the future we will be interested in the albedo representation by the model. We aim to use a coupled version with the atmospheric model ICON. Will there be large albedo perturbations if we set the SWE max to 1m with CLM5?

**Reply by billsacks** [reply]
My understanding is that the impact on albedo is very small, except in cases where 1 m snow depth is not enough to maintain a year-round snow pack in areas where there should be one, but I think that's relatively uncommon. However, I don't know that for sure.

**Reply by AdrienDams** [reply]
Thanks again, I have submit some cases with the new tags and the GCWP3 forcings. I will report back later.

**Comment by AdrienDams** [comment, marked as answer]
I just want to give an update, we have done a new simulation after a 30 years spin-up while using `reset_snow = .true.` and `h2osno_max = 800`. This give us a much better snow cover to use for our case (see below). Thank you all for your help.

![001_snow_depth_1980-02-212](https://user-images.githubusercontent.com/24732655/205075308-eaa5d293-6162-47a3-b33f-c4a61230fd51.png)
